### PR TITLE
Fix/unregister hs

### PR DESCRIPTION
**Pull request by elizabeth-jimenez**
Hopefully this helps with revoking Cory's overlay PR, just to make it easier to visualize.

### CODE REVIEW

File: src/models/bids.js [hunk 1]
```diff
@@ -30,55 +30,61 @@ const Bids = bookshelf.model('Bids', {
...
+          const available_position = values[0]
+          const employee = values[1]
+          const position = available_position.related('position')
+          const at_grade = position.get('pos_grade_code') === employee.get('grade_code')
+          const skill = position.related('skill')
+          const in_skill = employee._has_skill_code(skill.get('skl_code'))
+          const at_grade_in_skill = at_grade && in_skill
 
-        const bidstats = available_position.related('bidstats')
-        let cp_ttl_bidder_qty = bidstats.get('cp_ttl_bidder_qty')
```
Comment (elizabeth-jimenez) [inline src/models/bids.js:43]: ugh!!! indents!

File: src/models/bids.js [hunk 2]
```diff
@@ -74,11 +74,17 @@ const Bids = bookshelf.model('Bids', {
         // Position has been filled
         available_position.set('cp_status', 'FP')
         return available_position.save()
-      } else if (bs_cd === 'A' && handshakeOffered === 'Y') {
-        // Handshake has been offered on the position
-        available_position.set('cp_status', 'HS')
-        return available_position.save()
-      } 
+      } else if (bs_cd === 'A') {
+        if(handshakeOffered === 'Y') {
```
Comment (scott062) [inline src/models/bids.js:78]: add space :)

Comment (elizabeth-jimenez) [reply src/models/bids.js]: 👀

**Review by scott062 — APPROVED** [review]
Tested in insomnia 👍

**Pull request merged**